lab_1/main.py

```diff
@@ -1,42 +1,35 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import re
+def tokenize(text: str) -> list:
+    if isinstance(text,str): #проверяем, строка ли это
+        return []
+    text = re.sub(r'[^A-Za-z 0-9]', '', text) # с помощью рег. выраж. убираю знаки препинания и числительные
+    tokenize_list = text.split().lower()
+    return tokenize_list
+pass
 
 
-def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-    pass
 
 
 def remove_stop_words(tokens: list, stop_words: list) -> list:
-    """
```

Comment:
Please, leave the description of the function how it is - use the already written template.